lib/3scale/backend/transactor/status.rb

```diff
@@ -211,11 +194,40 @@ def with_report_and_hierarchy(reports)
           end
         end
 
-        def add_plan(xml, tag, plan_name)
+        def add_plan_section(xml, tag, plan_name)
           xml << "<#{tag}>"
           xml << plan_name.to_s.encode(xml: :text) << "</#{tag}>"
         end
 
+        def add_authorize_section(xml)
+          xml << '<authorized>'.freeze
+          if authorized?
+            xml << 'true'.freeze << '</authorized>'.freeze
```

Comment:
you can make this perform better with a single `xml << '<authorized>true</authorized>'.freeze`

Reply:
:+1: